content/millennium/r4/base/individuals/related-person.md

```diff
@@ -193,6 +193,33 @@ _Implementation Notes_
 
 #### Response
 
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Fri, 16 Jul 2021 17:23:14 GMT
+Etag: W/"0"
+Location: https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/RelatedPerson/204989530-12724068
+Last-Modified: Fri, 16 Jul 2021 17:23:13 GMT
+Vary: Origin
+X-Request-Id: 11111111111111111111111111111111
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+### Example- Encounter-Level RelatedPerson
+
+#### Request
+
+    POST https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/RelatedPerson
+
+#### Body
+
+<%= json(:r4_relatedperson_encounter_create) %>
+
+#### Response
+
```

Comment:
A general comment:  I think for the Body Field, the 'Required' column for the 'Related Person Encounter' should be changed to 'No' , as it is no more mandatory.

Reply:
+1

Reply:
updated: https://github.com/cerner/fhir.cerner.com/pull/689/commits/6ef20fc0201eeb01007296ad539296c4bcf3250e#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R204